app/views/_init-count.html

```diff
@@ -0,0 +1,11 @@
+<!--
+  Expects the following variables:
+  podTemplate
+ -->
+
+<span ng-if="pod.spec.initContainers.length" class="text-muted small">
+  <ng-pluralize count="pod.spec.initContainers.length"
+                when="{'1': '&nbsp;Init container','other': '&nbsp;{} Init containers'}">
+  </ng-pluralize>
+  <a ng-href="{{pod | navigateToTabURL:'details'}}">View Details</a>
```

Comment:
You need to pass `apiObject` in from the list row and pass that to the navigateToTabURL filter. Then check `apiObject.kind` to see what tab it should be. For `DeploymentConfig` and `Deployment`, you should use `configuration`. (I think, need to double check.)